metricbeat/docs/modules/haproxy.asciidoc

```diff
@@ -43,7 +43,7 @@ The default metricsets are `info`and `stat`.
 [float]
 === Compatibility
 
-The HAProxy metricsets were tested with HAProxy 1.6 and are expected to work with all 1.6 versions.
+The HAProxy metricsets are tested with HAProxy versions from 1.6, 1.7 to 1.8.
```

Comment:
Should we find a way to define the supported versions in fields.yml so we can generate this?

Reply:
We could have a central place where to define the versions matrix for the tests and use it also to generate doc, yes, this will require further thinking.

Reply:
I wonder if it would be enough if we start with an array `version: ["1.7", "2.1", "5.2-alpha"] or if this will not be enough, as we should also have the supported OS in there and for some versions only a subset of metricset works? Perhaps we should also allow to overwrite these values per metricset?